egui_extras/src/datepicker/button.rs

```diff
@@ -18,7 +18,7 @@ pub struct DatePickerButton<'a> {
 }
 
 impl<'a> DatePickerButton<'a> {
-    pub fn new(selection: &'a mut Date<Utc>) -> Self {
+    pub fn new(selection: &'a mut DateTime<Utc>) -> Self {
```

Comment:
I'm worried that this is a confusing API. Why would an API that is about picking a date (without time) take an argument for selecting the data _and time_?

I think perhaps there is a chance here for a better fix: to stop using `chrono::Date` in favor of our own date type, `pub struct Date { pub year: u16, pub month: u8, pub day: u8 }` or whatever. That way we don't force the use of `chrono` (a non-maintained crate with several open RUSTSEC:s) onto our users.